### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by igamasayuki**

### CODE REVIEW

File: src/main/java/com/example/repository/AdministratorRepository.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public class AdministratorRepository {
 	 * @throws EmptyDataAccessException 存在しない場合は例外を発生します
 	 */
 	public Administrator load(Integer id) {
-		String sql = "select id,name,mail_address,password from administrators where id=:id";
+			String sql = "select id,name,mail_address,password from administrators where id=:id";
```
Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 109).

Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 24, expected level should be 4.

File: src/main/java/com/example/repository/AdministratorRepository.java [hunk 2]
```diff
@@ -62,7 +62,8 @@ public Administrator findByMailAddressAndPassward(String mailAddress, String pas
 		SqlParameterSource param = new MapSqlParameterSource().addValue("mailAddress", mailAddress).addValue("password",
 				password);
 		List<Administrator> administratorList = template.query(sql, param, ADMINISTRATOR_ROW_MAPPER);
-		if (administratorList.size() == 0) {
+		if (administratorList.size() == 0)
```
Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/repository/AdministratorRepository.java [hunk 3]
```diff
@@ -62,7 +62,8 @@ public Administrator findByMailAddressAndPassward(String mailAddress, String pas
 		SqlParameterSource param = new MapSqlParameterSource().addValue("mailAddress", mailAddress).addValue("password",
 				password);
 		List<Administrator> administratorList = template.query(sql, param, ADMINISTRATOR_ROW_MAPPER);
-		if (administratorList.size() == 0) {
+		if (administratorList.size() == 0) 
+		{
```
Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.blocks.LeftCurlyCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'{' at column 3 should be on the previous line.

Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if lcurly' has incorrect indentation level 16, expected level should be 6.

File: src/main/java/com/example/repository/AdministratorRepository.java [hunk 4]
```diff
@@ -73,8 +74,9 @@ public Administrator findByMailAddressAndPassward(String mailAddress, String pas
 	 * 
 	 * @param administrator 管理者情報
 	 */
-	public void insert(Administrator administrator) {
-		SqlParameterSource param = new BeanPropertySqlParameterSource(administrator);
+	public void insert(Administrator administrator)
```
Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:77]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:77]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/example/repository/AdministratorRepository.java [hunk 5]
```diff
@@ -73,8 +74,9 @@ public Administrator findByMailAddressAndPassward(String mailAddress, String pas
 	 * 
 	 * @param administrator 管理者情報
 	 */
-	public void insert(Administrator administrator) {
-		SqlParameterSource param = new BeanPropertySqlParameterSource(administrator);
+	public void insert(Administrator administrator) 
+	{
```
Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:78]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:78]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.blocks.LeftCurlyCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'{' at column 2 should be on the previous line.

Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:78]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def lcurly' has incorrect indentation level 8, expected level should be 4.

File: src/main/java/com/example/repository/AdministratorRepository.java [hunk 6]
```diff
@@ -73,8 +74,9 @@ public Administrator findByMailAddressAndPassward(String mailAddress, String pas
...
+	SqlParameterSource param = new BeanPropertySqlParameterSource(administrator);
```
Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:79]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (igamasayuki) [inline src/main/java/com/example/repository/AdministratorRepository.java:79]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 8, expected level should be one of the following: 4, 6.

**Review by runy-iga1 — APPROVED** [review]
承認

**Pull request merged**